examples/cm6-graphql-parcel/yarn.lock

```diff
@@ -0,0 +1,2329 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
These should be git ignored, yes?

Reply:
`yarn.lock` should be included in git for version control

Reply:
We already have yarn.lock at the root for all package workspaces. This is an example. We don’t want the example lockfiles because they will trigger things like dependabot or security alerts that are not applicable

Reply:
Yeah I'll remove that.